### DISCUSSION TITLE
ArgoCD deployment dependencies

### DISCUSSION THREAD

**Discussion by saikumar13** [argoproj/argo-cd#8323] [General]
Hello,

Env details.
Kubelet: v1.11.3
helm: 2.12.1v
argo-cd: Rel.Version : XX

How to identify compatibility of the argo-cd particular release version with respective Kubernetes and helm version. Where can I find the minimum requirements with a particular release of argo-cd.

**Comment by samyak-jn** [comment]
Hi @saikumar13,

I think this #4110 issue is up that talks about similar want, and for doc references check [here](https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/overview/), kindly take a look. Meanwhile, I'll get back to you with a concrete answer or a reference, if I found one.

Helm: https://argo-cd.readthedocs.io/en/stable/user-guide/helm/#helm-version
Kubernetes: The project support all the active version of kubernetes
